content/post/2020-06-01-macos-installer/index.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+slug: macos-installer
+title: "Using muliple versions of R on macOS"
+authors:
+  - Gábor Csárdi
+date: "2020-06-01"
+tags:
+- package development
+- installer
+- R versions
+output: hugodown::hugo_document
+---
+
+<!-- -*- mode: gfm; canonical: true -*- -->
+
+```{r include = FALSE}
```

Comment:
Could you please change knitr options so that gifs are directly in the post folder? I.e. index.Rmd, index.md and all gifs at the same "level".

Reply:
https://github.com/r-hub/blog/blob/c0c7a9583d55a4d16c95ccd91659d5ccbbe4a68d/content/post/2019-12-19-r-hub-usage/index.Rmd#L15